Fixed Background on Sock.tsx

roslyn-maloney opened the pull request:
# Added a SVG file for the sock background

## 🎫 Issue #293.

### ▶ Changelist:
- Created a new TSX file in ./lib/Assets/SVG/OurTeamPageAssets/ called JoinCoreBackground.tsx and pasted the converted SVG to React code in.
- Replace AboutUs.png with JoinCoreBackground.tsx.
- Replace instances of on the Sock.tsx file for OurTeam with the new asset
- Updated formatting/layout of the screen in respect to the new background

### 📝 Notes + 🚧 TODO:

- There was a weird spacing between the sock and footer once I updated the background of the sock
- Updated the sock to have z-index of 10 so that it will be behind the footer and increased the desktop aspect
- Also made sure there was no padding around the background

### 🧪 Testing:

- Tested format on Tablet, Phone, and Desktop (SS below)

### 🎥 Screenshots & Screencasts:

Desktop:
<img width="1510" height="861" alt="Screenshot 2025-11-03 at 10 57 46 AM" src="https://github.com/user-attachments/assets/471f545b-3347-4999-a3bb-1e2935234cd5" />

Tablet:
<img width="1180" height="820" alt="IMG_0158" src="https://github.com/user-attachments/assets/aaedeb0e-f37d-42f9-b742-0f0fb9bb5295" />

Phone: 
<img width="590" height="1278" alt="IMG_9770" src="https://github.com/user-attachments/assets/08ce2d84-2d2d-429b-a6ba-ea5b71bb4a1b" />

roslyn-maloney committed:
fixed background on sock.tsx

agonzalez22 commented on apps/main/src/app/team/Sections/Sock.tsx:
```diff
@@ -3,8 +3,7 @@
 
-import Image from "next/image";
-import AboutUs from "../../lib/Assets/SVG/OurTeamAssets/AboutUs.png";
+import AboutUsBackground from "../../lib/Assets/SVG/OurTeamAssets/JoinCoreBackground.tsx";
```
Names should match--- keep as JoinCoreBackground instead of AboutUsBackground

agonzalez22 commented on apps/main/src/app/team/Sections/Sock.tsx:
```diff
@@ -17,18 +16,15 @@ import AboutUs from "../../lib/Assets/SVG/OurTeamAssets/AboutUs.png";
 const Sock = () => {
   return (
     <div
```
<img width="944" height="485" alt="Image" src="https://github.com/user-attachments/assets/1f091c5d-fdaa-42e3-ba4c-8fdf643a0927" />

<img width="943" height="371" alt="Image" src="https://github.com/user-attachments/assets/1b23b807-ba0a-4528-8def-0fddc386b79f" />

<img width="1135" height="445" alt="Image" src="https://github.com/user-attachments/assets/3c91c198-9c0e-4348-b022-e0568d5d7007" />

on wider screen sizes this is what happens, I added a ss of my whole screen so you can replicate locally if need be. 

<img width="1899" height="907" alt="Image" src="https://github.com/user-attachments/assets/3f629cb9-036b-46ee-a24f-92fca8c84821" />

agonzalez22 replied:
Might want to mess around with aspect on larger screen sizes or look into `preserveAspectRatio="none"`

agonzalez22 reviewed (changes requested):
hehe gj gj

roslyn-maloney committed:
fixed spacing and sizing for sock on larger screen

roslyn-maloney commented:
# Added a SVG file for the sock background + Fixed spacing on larger screens

## 🎫 Issue #293.

### ▶ Changelist:
- Created a new TSX file in ./lib/Assets/SVG/OurTeamPageAssets/ called JoinCoreBackground.tsx and pasted the converted SVG to React code in.
- Replace AboutUs.png with JoinCoreBackground.tsx.
- Replace instances of on the Sock.tsx file for OurTeam with the new asset
- Updated formatting/layout of the screen in respect to the new background
- Used preserveAspectRatio="none" on the background and moved the footer closer to the sock using margintop (-8)

### 📝 Notes + 🚧 TODO:

- There was a weird spacing between the sock and footer once I updated the background of the sock
- Updated the sock to have z-index of 10 so that it will be behind the footer and increased the desktop aspect
- Also made sure there was no padding around the background
- Tested on all screens (mobile, 14in macbook, 16in macbook, and tablet)

### 🧪 Testing:

- Tested format on Tablet, Phone, and 14 + 16 inch Desktop (SS below)

### 🎥 Screenshots & Screencasts:

Desktop (14in):
<img width="1510" height="861" alt="Screenshot 2025-11-03 at 10 57 46 AM" src="https://github.com/user-attachments/assets/471f545b-3347-4999-a3bb-1e2935234cd5" />

Desktop (16inch):
<img width="3432" height="1988" alt="image" src="https://github.com/user-attachments/assets/b5160739-79cc-4f38-a639-3a3bcab68d45" />

Tablet:
<img width="1180" height="820" alt="IMG_0158" src="https://github.com/user-attachments/assets/aaedeb0e-f37d-42f9-b742-0f0fb9bb5295" />

Phone: 
<img width="590" height="1278" alt="IMG_9770" src="https://github.com/user-attachments/assets/08ce2d84-2d2d-429b-a6ba-ea5b71bb4a1b" />

roslyn-maloney committed:
fixed sizing of sock on larger screens using min-h

roslyn-maloney committed:
Added mobile view to the sock

roslyn-maloney commented:
# Added a SVG file for the sock background

## 🎫 Issue #293.

### ▶ Changelist:
- Created a new TSX file in ./lib/Assets/SVG/OurTeamPageAssets/ called JoinCoreBackground.tsx and pasted the converted SVG to React code in.
- Replace AboutUs.png with JoinCoreBackground.tsx.
- Replace instances of on the Sock.tsx file for OurTeam with the new asset
- Updated formatting/layout of the screen in respect to the new background

### 📝 Notes + 🚧 TODO:

- used min-h and perspectiveAspectRatio: none to get rid of unnecessary spacing on larger screens

### 🧪 Testing:

- Tested format on Tablet, Phone, and Desktop (SS below)

### 🎥 Screenshots & Screencasts:

Desktop (14inch):
<img width="1510" height="861" alt="Screenshot 2025-11-03 at 10 57 46 AM" src="https://github.com/user-attachments/assets/471f545b-3347-4999-a3bb-1e2935234cd5" />

Desktop (16inch):
<img width="3432" height="1988" alt="image" src="https://github.com/user-attachments/assets/b5160739-79cc-4f38-a639-3a3bcab68d45" />

Desktop (24inch)
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/23fe1be6-22dd-4669-9730-2135bca4d3ae" />

Tablet:
<img width="1180" height="820" alt="IMG_0158" src="https://github.com/user-attachments/assets/aaedeb0e-f37d-42f9-b742-0f0fb9bb5295" />

Phone: 
<img width="590" height="1278" alt="IMG_9770" src="https://github.com/user-attachments/assets/08ce2d84-2d2d-429b-a6ba-ea5b71bb4a1b" />

agonzalez22 commented on apps/main/src/app/team/Sections/Sock.tsx:
```diff
@@ -17,18 +16,12 @@ import AboutUs from "../../lib/Assets/SVG/OurTeamAssets/AboutUs.png";
-      />
+      <div className="absolute">
+        <JoinCoreBackground preserveAspectRatio="none"/>
+      </div>
```
<img width="1912" height="721" alt="Image" src="https://github.com/user-attachments/assets/037ecdd4-b737-4587-99fc-a6585088df0c" />

getting this now on larger screen sizes but the previous issues have resolved

roslyn-maloney committed:
fixed offsetting on larger screens

roslyn-maloney commented:
# Added a SVG file for the sock background

## 🎫 Issue #293.

### ▶ Changelist:
- Created a new TSX file in ./lib/Assets/SVG/OurTeamPageAssets/ called JoinCoreBackground.tsx and pasted the converted SVG to React code in. Got rid of the fixed height and width in the TSX file
- Replace AboutUs.png with JoinCoreBackground.tsx.
- Replace instances of on the Sock.tsx file for OurTeam with the new asset
- Updated formatting/layout of the screen in respect to the new background

### 📝 Notes + 🚧 TODO:

- There was a weird spacing between the sock and footer once I updated the background of the sock
- Updated the sock to take up the whole screen width and for it be center screen
- Also add bottom padding so that there is no weird spaces on smaller screens

### 🧪 Testing:

- Tested format using chromeDevtools
- Made the width 5000, 3000, 2000, and 1000
- Also tested on Phone, tablet and computer

### 🎥 Screenshots & Screencasts:

Desktop:
<img width="1510" height="861" alt="Screenshot 2025-11-03 at 10 57 46 AM" src="https://github.com/user-attachments/assets/471f545b-3347-4999-a3bb-1e2935234cd5" />

Tablet:
<img width="1180" height="820" alt="IMG_0158" src="https://github.com/user-attachments/assets/aaedeb0e-f37d-42f9-b742-0f0fb9bb5295" />

Phone: 
<img width="590" height="1278" alt="IMG_9770" src="https://github.com/user-attachments/assets/08ce2d84-2d2d-429b-a6ba-ea5b71bb4a1b" />

5000 Width
<img width="1511" height="773" alt="image" src="https://github.com/user-attachments/assets/8790c027-faef-46f6-8254-97aa30008ec6" />

3000 Width
<img width="1511" height="773" alt="image" src="https://github.com/user-attachments/assets/8a5fcaa7-fef9-4d81-b5c2-01dce1a1a513" />

2000 Width
<img width="1511" height="773" alt="image" src="https://github.com/user-attachments/assets/d4aef41c-3287-4fa0-8cc9-618266477302" />

1000 Width
<img width="1064" height="773" alt="image" src="https://github.com/user-attachments/assets/4c1d5467-60f6-467c-9968-4b8fd14f6690" />

agonzalez22 committed:
Merge branch 'dev' into mainsite/bug-fix/our-team-sock-background

agonzalez22 reviewed (approved):
fantastic job!
